prime-router/metadata/fhir_mapping/hl7/segments/MSH/Provenance-Source.yml

```diff
@@ -1,3 +1,8 @@
+# Entity.role is required if SFT exists. The mapping inventory has the following direction:
+#   - If the software does not represent the original source system then Entity.role=derivation
+#   - If the software does represent the original source system then Entity.role=source
+# Either source or derivation will need to be set for each sender
```

Comment:
Comment is a little confusing since we're hardcoding to "source"... is this saying that we would use a sender transform to change it?

Reply:
Oh yep, forgot about this. Removed this comment and added info to HL7v2-FHIR-Inventory.md